core/src/main/java/org/jdbi/v3/BuiltInArgumentFactory.java

```diff
@@ -36,14 +41,14 @@
  * class directly, but instead should bind your object with the
  * {@link SQLStatement} convenience methods.
  */
-public class BuiltInArgumentFactory<T> implements ArgumentFactory<T> {
+public class BuiltInArgumentFactory implements ArgumentFactory<Object> {
```

Comment:
In this case I didn't like how the `<T>` parameter just morphed into whatever we decided it was during the method call.

I actually think we should get rid of the parameter on `ArgumentFactory` like on the other factories--although I was planning to do that as part of the factory method coalescing I mentioned elsewhere on this ticket.